lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -1150,6 +1191,180 @@ void SESERegionBuilder::ensureSingleExitFromLoops() {
   }
 }
 
+void SingleExitLoopTransformer::unrollLoopBodyOnce() {
+  // Consider the following example:
+  //
+  //   do {
+  //     i += 1
+  //     if (...) break;
+  //     i += 1
```

Comment:
the example could be more readable if we use sth different from "i += 1" here, say "i += 2"

Reply:
Good idea. Done.